pkg/model/dev.go

```diff
@@ -449,6 +459,10 @@ func (dev *Dev) ToTranslationRule(main *Dev) *TranslationRule {
 				Name:  oktetoMarkerPathVariable,
 				Value: path.Join(dev.MountPath, dev.DevPath),
 			},
+			EnvVar{
```

Comment:
Could you add the envvar only if `SSHServerPort` is different than 22?
This way we modify the original deployment the minimum possible

Reply:
Agree — fixed.